Cisco/cisco-meraki/_meta/smart-descriptions.json

```
@@ -202,7 +202,7 @@
         ]
     },
     {
-        "value": "dhcp request for {client.ip} from {client.mac}",
+        "value": "dhcp request for {source.ip} from {source.mac}",
```

```suggestion
        "value": "DHCP offer for {source.ip} to {source.mac}",
```

:warning: a DHCP offer is emitted by the DHCP server for the client (the server gives an IP chosen by himself for the client)